actions/versions.go

```diff
@@ -0,0 +1,33 @@
+package actions
+
+import (
+	"sort"
+	"strings"
+)
+
+// getVersions returns a list of all versions under a given prefix
+// This assumes a specific format when organizing the objects in the S3 bucket, e.g.
+//   s3://bucket/prefix/1.0/file.json
+func getVersions(prefix, delimiter string) ([]string, error) {
+	dv, err := S3.ListObjects(prefix, delimiter)
+	if err != nil {
+		return nil, err
+	}
+
+	var versions []string
+	// split the object prefixes to get the last part which is the version
+	for _, p := range dv.CommonPrefixes {
+		s := strings.Split(strings.Trim(*p.Prefix, "/"), "/")
+		versions = append(versions, s[len(s)-1])
+	}
+	return versions, nil
+}
```

Comment:
Shouldn't this just be a method on `RiskLevels`?

Reply:
It's also used for getting the question versions.

Reply:
Ah! perfect! your first interface! 😆